src/content/guides/prepare-aws-account-for-tenant-clusters/index.md

```diff
@@ -64,7 +64,7 @@ These are the limit increases to be requested, grouped by limit type:
   - Auto Scaling Groups per region: **250**
   - Launch Configurations per region: **500**
 - EC2 Instances
-  - m3.large per region: **250**
```

Comment:
@calvix I think in the past at some point `m3.large` was a default instance type. Now the default is `m4.xlarge`.

Can you make sure there is no type missing which we use frequently?

Reply:
tbh this heavily depends on customers instance type

Reply:
OK. I added an extra sentence about that.